### ISSUE TITLE
Add `containedInPlace` to ResponseType `Place`

### ISSUE THREAD

**Issue by HenKun** [netresearch/dhl-sdk-api-unified-location-finder#1]
According to [API data schema](https://developer.dhl.com/api-reference/location-finder#reference-docs-section) there is a property `containedInPlace` inside the `place` property, besides `geo` and `address`. This describes as follows: "Description of the building/facility etc. this service point is located within". This can be a valuable hint for a customer where to find the pickup location, especially for Packstations that do not have descriptive names. First, the customer can find the place when he does not know where the Packstation is exactly located. Second, the customer might know "My Packstation is at Edeka Supermarket", but does not know which Packstation number he has to choose.

Especially for Packstations that are located within train stations or shopping centers that could be quite helpful.

Example: 
1. Search for postal code `09119` in country `DE`
2. Find location `8007-409119139` which is "Packstation 139"
3. See that there is no descriptive name for this Packstation, besides "Packstation 139".
4. The `containedInPlace` property, in this case "EDEKA Markt" gives the customer a hint, where the Packstation is located and he can choose the correct Packstation without knowing the Packstation number or the street address.

**Comment by mam08ixo** [comment]
Thank you @HenKun, that sounds like a useful improvement. The property was introduced in API version 1.6.0 after the library was created. I cannot give you an ETA but I do see this implemented in a future version of the package.

**Comment by HenKun** [comment]
Thanks for considering. I modified the existing code to already include the property on my website, and it is beautiful. For your help, what I found out additionally:

1. `containedInPlace->name` might be a misleading name, it is more a broad description of where the Packstation can be found (near / in / at /by), see the image. So one does not need to assume the preposition oneself, DHL provides the preposition itself. This is really helpful. I do not know if the phrases are localized, I only tested for German language. 
![dhl-contained-in-place](https://user-images.githubusercontent.com/18382633/192092418-03e43545-c664-4e4b-bfe1-c29a31229fd2.png)
2. It is not given for every Packstation. As far as I could see it is never given for parcel shops or post offices, but not sure about that, since in theory a parcel shop or post office could be located within a larger shopping centre.
3. If not available, the `containedInPlace` property is not empty, it is not present in the dataset at all.
